conda_forge_tick/make_graph.py

```diff
@@ -12,68 +14,72 @@
 
 
 def get_attrs(name, i):
-    sub_graph = {
-        "time": time.time(),
-        "feedstock_name": name,
-        # All feedstocks start out as good
-        "bad": False,
-    }
-
-    logger.info((i, name))
-    r = requests.get(
-        "https://raw.githubusercontent.com/"
-        "conda-forge/{}-feedstock/master/recipe/"
-        "meta.yaml".format(name)
-    )
-    if r.status_code != 200:
-        logger.warn(
-            "Something odd happened when fetching recipe "
-            "{}: {}".format(name, r.status_code)
+    try:
+        sub_graph = {
```

Comment:
try-blocks should be as minimal as possible. Could you maybe move this to a helper function please? Or become more targeted in where the try-excepts are. Thanks!